src/cdk/stepper/stepper.ts

```diff
@@ -126,6 +156,20 @@ export class CdkStep implements OnChanges {
     return this.stepControl ? this.stepControl.valid && this.interacted : this.interacted;
   }
 
+  /** Whether step has error. */
+  @Input()
+  get hasError(): boolean {
+    return this._customError == null ? this._getDefaultError : this._customError;
```

Comment:
`===` is preferred to `==`

If you just want to do a truthyness check in this case, you could use:

```ts
return this._customError || this._getDefaultError;
```